### PR TITLE
Fix babel transpiling issue on models

### PR DESCRIPTION
**Pull request by jingyi-gao-ttd**
Manual testing 

https://github.com/IABTechLab/uid2-self-serve-portal/assets/116539741/fc9f0627-a187-405c-af77-0edaba29f504

### CODE REVIEW

File: .storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
```
Comment (alex-yau-ttd) [inline .storybook/main.ts:1]: Why do we need this change? Is it because of the later `@babel/core` version or something else?

**Review by alex-yau-ttd — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -93,7 +93,8 @@
     "start": "craco start",
     "build": "craco build",
     "test": "craco test --testPathPattern=src/web/",
-    "api": "nodemon -r dotenv/config ./src/api/api.ts",
+    "api": "nodemon -e \".ts\" --exec npm run babel-node-api",
+    "babel-node-api": "babel-node -r dotenv/config -x \".ts\" ./src/api/api.ts",
```
Comment (alex-yau-ttd) [inline package.json:97]: What does this command actually do? Should we update the README?

Comment (alex-yau-ttd) [reply package.json:97]: oh right, `api` calls this, so I guess no README changes needed.

File: package.json [hunk 3]
```diff
@@ -93,7 +93,8 @@
     "start": "craco start",
     "build": "craco build",
     "test": "craco test --testPathPattern=src/web/",
-    "api": "nodemon -r dotenv/config ./src/api/api.ts",
+    "api": "nodemon -e \".ts\" --exec npm run babel-node-api",
```
Comment (alex-yau-ttd) [inline package.json:96]: can you explain the actual difference this makes? Does it just ensure that it pulls in the babel config when running?

Comment (jingyi-gao-ttd) [reply package.json:96]: Yes, babel-node-api will use babel-node to transpile the file under api

**Review by lionell-pack-ttd — APPROVED** [review]

**Pull request merged**